cmd/criticize/main.go

```diff
@@ -33,24 +34,18 @@ type linter struct {
 
 	// Command line flags:
 
-	configFile string
-
-	withOpinionated    bool
-	withExperimental   bool
-	checkGenerated     bool
-	shorterErrLocation bool
+	fp *flagparser.FlagParser
```

Comment:
s/fp/flags ?

Reply:
Done, but why it is better? Is it objective, or subjective?